frontend/src/container/AdminAccount.jsx

```diff
@@ -450,10 +451,11 @@ export class Account extends React.Component {
   setHeadTitle = () => {
     const { props, state } = this
     if (props.system.config.instance_name && state.userToEdit.publicName) {
-      GLOBAL_dispatchEvent({
-        type: CUSTOM_EVENT.SET_HEAD_TITLE,
-        data: { title: buildHeadTitle([this.props.t('User administration'), state.userToEdit.publicName, props.system.config.instance_name]) }
-      })
+      const headTitle = buildHeadTitle(
+        [this.props.t('User administration'), state.userToEdit.publicName, props.system.config.instance_name]
+      )
+
+      props.dispatch(setHeadTitle(headTitle))
     }
```

Comment:
This kind of thing too, by the way. Maybe we should create a base `TracimReactComponent` class, or something?